### PR TITLE
Upgrade Steal to latest

### PR DESCRIPTION
**Pull request by marshallswain**

### CODE REVIEW

**Comment by phillipskevin** [comment]
Need to update the build script also. https://github.com/canjs/can-compute/blob/master/build.js#L4

**Comment by marshallswain** [comment]
Oh thanks, @phillipskevin!  Done.

File: package.json [hunk 1]
```diff
@@ -29,8 +29,10 @@
     "canjs-plugin",
     "donejs"
   ],
-  "system": {
-    "npmAlgorithm": "flat"
+  "steal": {
+    "plugins": [
+      "steal-qunit"
```
Comment (phillipskevin) [inline package.json]: I don't think this is necessary.

Comment (marshallswain) [inline package.json]: Oh.  It's imported manually, right?

**Comment by phillipskevin** [comment]
👍  looks good.

**Pull request merged**